app/controllers/users_controller.rb

```diff
@@ -3,4 +3,13 @@ def show
     @user = current_user
     @user_characters = @user.characters.order(:name).page(params[:page]).per(3)
   end
+
+  def destroy
+    @user = User.find(params[:id])
+    redirect_to(root_url, notice: "User deleted.") if @user.destroy
+  end
+
+  def profile
+    @user = User.find(params[:id])
```

Comment:
@zzokki81 again does this action allow me to get info about any user? Also, now that I think about it I don't see that you are performing any user authentication - are you doing that somewhere (i.e. checking if the user is signed in)?